examples/basics/visualising_open_industrial_data.ipynb

```diff
@@ -0,0 +1,436 @@
+   "outputs": [],
+   "source": [
+    "client.assets.retrieve(id=2853212781345885).to_pandas()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "YIejfGjYWwNY"
+   },
+   "source": [
+    "#### View datapoints for one of the time series\n",
+    "\n",
+    "* Now that we have a list of time series, we can retrieve datapoints for some of the them.\n",
+    "\n",
+    "* The returned object is a data frame with\n",
+    "\n",
+    "  * timestamp which is time in milliseconds as index\n",
+    "  \n",
+    "  * value which is the reading from the sensor as columns\n",
+    "  \n",
+    "The identifier to retrieve Datapoints is the **id** or the external id equal to the **name** column from the DataFrame above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "i_3nqZxPK0KZ"
+   },
+   "outputs": [],
+   "source": [
+    "client.datapoints.retrieve(external_id=\"VAL_45-PT-92508:X.Value\", start=\"10d-ago\", end=\"now\").to_pandas().head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "S3fBvnnKL3ii"
+   },
+   "source": [
+    "## Visualising the data in the CDF\n",
+    "\n",
+    "* The rest of the notebook will be used to retrieve and visualize data\n",
+    "\n",
+    "* The sensor are chosen because they are specifically good if you later want to try to create a model that predicts one sensor from the others."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "mLJBe0WuWKDq"
+   },
+   "source": [
+    "#### Collect datapoints from CDF\n",
+    "\n",
+    "* The following four time series measures the input flow rate, pressure, and temperature and the output pressure from a suction cooler\n",
+    "\n",
+    "  * FT: Input Flow Rate\n",
+    "  \n",
+    "  * PT: Input Pressure\n",
+    "  \n",
+    "  * TT: Input Temperature\n",
+    "\n",
+    "  * Output Pressure\n",
+    "\n",
+    "  \n",
+    "The time series names are defined in the *in_ts_names* and *out_ts_names* lists below.\n",
+    "  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "l4-C138i-HBy"
+   },
+   "outputs": [],
+   "source": [
+    "in_ts_names = [\"VAL_23-FT-92512:X.Value\", \"VAL_23-PT-92512:X.Value\", \"VAL_23-TT-92502:X.Value\"]\n",
+    "out_ts_name = \"VAL_23-PT-92504:X.Value\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "lvr3qJp_OQgF"
+   },
+   "source": [
+    "#### Retrieve Data Points from CDF\n",
+    "\n",
+    "* Datapoints are returned in a pandas dataframe using the method *datapoints.retrieve()*, [documentation](https://cognite-docs.readthedocs-hosted.com/projects/cognite-sdk-python/en/latest/cognite.html#retrieve-multiple-time-series-by-id).\n",
```

Comment:
.retrieve() does not return a pandas DataFrame, it returns Union[Datapoints, DatapointsList]. .to_pandas() returns a pd.DataFrame.

Reply:
I'll change the text.